documentation/modules/deploying/proc-deploying-kafkaconnector.adoc

```diff
@@ -53,7 +53,7 @@ spec:
 <3> Full name or alias of the connector class. This should be present in the image being used by the Kafka Connect cluster.
 <4> Maximum number of Kafka Connect `Tasks` that the connector can create.
 <5> xref:#kafkaconnector-configs[Connector configuration] as key-value pairs.
-<6> Replace `/opt/kafka/LICENSE` with a temporary file to read the messages from. 
+<6> The source connector reads data from the `/opt/kafka/LICENSE` file. 
```

Comment:
Perhaps

"This example source connector configuration reads data from the `/opt/kafka/LICENSE` file."

?

Reply:
Agreed. I've changed the wording.